app/uk/gov/hmrc/ngrpropertylinkingfrontend/views/SingleSearchResultView.scala.html

```diff
@@ -45,7 +45,7 @@
     <p class="govuk-body">
         @messages("singleSearchResultPage.details.content.2") <br>
         @messages("singleSearchResultPage.details.content.3")
-        <a href= @searchAgainUrl class="govuk-link">@messages("singleSearchResultPage.details.link")</a><p>
+        <a href= @searchAgainUrl class="govuk-link">@messages("singleSearchResultPage.details.link").</a><p>
```

Comment:
I believe that period after Search again is outside of hyper link.
```suggestion
       @linkComponent(label = messages("singleSearchResultPage.details.link"), hrefUrl = searchAgainUrl, beforeLinkContent = messages("singleSearchResultPage.details.content.2")<br>messages("singleSearchResultPage.details.content.3"), afterLinkContent = ".")
```